base/uk.ac.stfc.isis.ibex.opis/resources/jaws/blades_script.py

```diff
@@ -52,23 +66,69 @@ def run(self):
                 return
 
             self.get_pvs()
-
+            
+            #Without the following code, if the horizontal and vertical limits of travel differ
+            #but the vertical and horizontal gaps are set equal, the gap on the GUI will not appear
+            #square as expected. The following applies a scaling factor to the jaw heights/widths to correct this
+            #and calculates x and y positions for the limit of travel indicators.
+            if self.max_x > self.max_y:
+                scaling_factor = self.max_y/self.max_x
+                self.north *= scaling_factor
+                self.south *= scaling_factor
+                north_bound_val = background_height*scaling_factor/2.0
+                south_bound_val = background_height - background_height*scaling_factor/2.0
```

Comment:
It might be worth pulling this out into a method with a bunch of doc strings like we did for `calc_pair_height`